pino.js

```diff
@@ -217,7 +224,7 @@ function fastRep (s) {
   } else {
     result += str.slice(last)
   }
-  return result
+  return point < 32 ? JSON.stringify(str) : '"' + result + '"'
```

Comment:
If the conditional is moved to break the loop whenever a code point below 32 is found, this just becomes:

```javascript
return '"' + result + '"'
```